languages/javascript/exercises/concept/errors/.docs/instructions.md

```diff
@@ -0,0 +1,95 @@
+Elena is the new quality manager of a newspaper factory. As she has just arrived in the company, she has decided to review some of the processes in the factory to see what could be improved. She found out that technicians are doing a lot of quality checks by hand. She sees there a good oportunity for automation and ask you, a freelance developer, to develop a piece of software to monitor some of the machines.
+
+## Check the humidity level of the production room
+
+Your first mission is to write a piece of software to monitor the humidity level of the production room. There is already a sensor connected to the software of the company that returns periodically the humidity percentage of the room. You need to implement a function in the software that will throw an error if the humidity percentage is too high.
+
+## Detect overheating and broken sensors
+
+Elena is very pleased with your first assignement and ask you to deal with the monitoring of the machines' temperature.
+
+While chatting with a technician, Greg, you are told that if the temperature of a machine exceeds 500°C, the technicians start worrying about overheating.
+
+The machine is equiped with a sensor that measures its internal temperature. You should know that the sensor is very sensitive and often breaks. In this case the technicians will need to change it.
+
+Your job is to implement a function that throws an error if the sensor is broken or if the machine starts overheating.
+
+As you know that you will later need to react differently depending on the error, you need a mechanism to differentiate the two kind of errors.
```

Comment:
Compare the 2 for what reads best:
**current**: As you know that you will later need to react differently
**suggested**: Knowing that you will later need to react different

Reply:
aaa7fe2a557b19ad60f0a2626d6e2f7b734d1f0c